README.rst

```diff
@@ -50,14 +50,26 @@ The built documentation will be available at::
 
   /absolute/path/to/cellml-specification/build/singlehtml/index.html
 
+Building PDFs
+^^^^^^^^^^^^^
+
 To build the PDF form of the documentation, simply execute the following command::
 
   make latexpdf
 
-**Note:** creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
-Also check that you have ``latexmk`` available as this is required for creating the PDF form of the documentation.
+If this fails, please have a look at the following:
+
+Creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
+
+In addition, you'll need the following latex packages:
```

Comment:
"LaTeX", not "latex".

Reply:
That's marketing!

Reply:
Having seen "LaTeX" all those years, "latex" looks really odd.

Reply:
Ok fixed it for you

Reply:
Woo hoo! Thanks @MichaelClerx. :)